pontoon/base/models.py

```diff
@@ -3425,3 +3432,13 @@ def calculate_stats(self, save=True):
             strings_with_warnings_diff,
             unreviewed_strings_diff,
         )
+
+
+class Comment(models.Model):
+    author = models.ForeignKey(User)
+    timestamp = models.DateTimeField(default=timezone.now)
+    translation = models.ForeignKey(Translation)
+    comment = models.TextField(blank=True)
+
+    def __unicode__(self):
+        return self.translation.string
```

Comment:
Why are you rendering a translation?

Reply:
This was actually another question I had and forgot to add. When accessing the model from `admin` this is what displays, without this it defaults to `Comment object` for all comments.  I went back and forth on whether this should be the translation or the comment, and settled on the translation since I figured that would be the reference that most would look for when accessing through the admin panel.

Reply:
I'd prefer `self.content` to make it consistent with the `Entity` and `Translation` admin.